stable/dex-controller/templates/deployments.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: {{ template "dex-controller.fullname" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    release: {{ .Release.name }}
+    heritage: {{ .Release.Service }}
+  name: {{ template "dex-controller.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: {{ template "dex-controller.fullname" . }}
+  template:
+    metadata:
+      labels:
+        control-plane: {{ template "dex-controller.fullname" . }}
+        chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+        release: {{ .Release.name }}
+        heritage: {{ .Release.Service }}
+    spec:
+      serviceAccountName: {{ template "dex-controller.fullname" . }}-sa
+      containers:
+      - args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=10
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.4.0
```

Comment:
I will make the image and tag configurable as properties. We do it in other charts

Reply:
Done.